src/bot/exts/fun/uwu.py

```diff
@@ -199,6 +60,6 @@ async def uwu_command(self, ctx: Context, *, text: t.Optional[str] = None) -> No
         await ctx.send(content=converted_text, embed=embed)
 
 
-async def setup(bot: Bot) -> None:
+async def setup(bot: commands.Bot) -> None:
```

Comment:
```suggestion
async def setup(bot: Bot) -> None:
```
This `Bot` is the subclass defined in `bot/bot.py`, not the base Discord.py `commands.bot`.